locale/pt-br/blog/announcements/nodejs-certified-developer-program.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Os Parceiros da Fundação Node.js com a Fundação Linux no Novo Programa de Certificação Node.js
+date: 2017-1-26T12:00:00.000Z
+category: Comunicados
+slug: nodejs-certified-developer-program.md
+layout: blog-post.hbs
+---
+
+_Fundação Node.js a ser lançada
+fornecedor-neutro programa de certificação para a plataforma que mais cresce_
+
+**SAN FRANCISCO, Jan. 26, 2017** — [A Fundação Node.js](https://foundation.nodejs.org/), um liderado pela comunidade e
```

Comment:
There's something wrong here, "um liderado pela ..." this translation asks for more information

Reply:
amazing @khaosdoctor, did you found some other phrase with conjugation wrong?
Or just this?